django/contrib/postgres/operations.py

```diff
@@ -261,3 +263,64 @@ def describe(self):
     @property
     def migration_name_fragment(self):
         return 'remove_collation_%s' % self.name.lower()
+
+
+class AddConstraintNotValid(AddConstraint):
+    """
+    Add a table constraint without enforcing validation, using PostgreSQL's
+    NOT VALID syntax.
+    """
+    def __init__(self, model_name, constraint):
+        if not isinstance(constraint, CheckConstraint):
+            raise NotSupportedError('AddConstraintNotValid does not support %s.' % (
+                constraint.__class__.__name__))
+        self.model_name = model_name
+        self.constraint = constraint
```

Comment:
Should this call `super().__init__(model_name, constraint)` instead of re-implementing the base class' logic?